src/ResourceManager/Compute.ManagedService/AzureRM.Compute.ManagedService.psd1

```diff
@@ -0,0 +1,126 @@
+#
+# Module manifest for module 'PSGet_AzureRM.Compute.ManagedService'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 01/19/2018
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+# RootModule = ''
```

Comment:
**REQUIRED:** You need to include a RootModule which is a psm1.  This file should be similar to the psm1 files used by our modules.  In this case you qill need it to:

(1) Check for the Hyper-V module and import it if it is available.  If it is not available, write an error with the links that were provided by the Hyper-V team for installation
(2) Otherwise, dot-source the psm1 file

Reply:
@markcowl Is it feasible for you help provide a more detailed example for us about what to do here to setup the script cmdlet properly? Or any reference to other existing modules in AzurePS?